drivers/place/bookings.cr

```diff
@@ -164,7 +164,11 @@ class Place::Bookings < PlaceOS::Driver
 
     self[:custom_qr_color] = setting?(String, :custom_qr_color)
     self[:custom_qr_url] = setting?(String, :custom_qr_url)
-    self[:show_qr_code] = !(setting?(Bool, :hide_qr_code) || false)
+
+    hide_qr_code = setting?(Bool, :hide_qr_code) || false
+    show_qr_code = setting?(Bool, :show_qr_code) || false
+
+    self[:show_qr_code] = !hide_qr_code || show_qr_code
```

Comment:
The settings both kind of clash
i.e. by default hide_qr_code == true, but if show_qr_code is set to false then the setting won't be applied.

```suggestion
    self[:show_qr_code] = show_qr_code.nil? ? !hide_qr_code : show_qr_code
```